CI/appveyor.after_success.ps1

```diff
@@ -8,11 +8,12 @@ windeployqt.exe --release mudlet.exe
 
 Remove-Item * -include *.cpp, *.o
 
-if ("$Env:APPVEYOR_REPO_TAG" -eq "false") {
-  $DEPLOY_URL = "https://ci.appveyor.com/api/buildjobs/$Env:APPVEYOR_JOB_ID/artifacts/src%2Fmudlet.zip"
-} else {
-  git clone https://github.com/Mudlet/installers.git C:\projects\installers
+#if ("$Env:APPVEYOR_REPO_TAG" -eq "false") {
+#  $DEPLOY_URL = "https://ci.appveyor.com/api/buildjobs/$Env:APPVEYOR_JOB_ID/artifacts/src%2Fmudlet.zip"
+#} else {
+  git clone https://github.com/keneanung/mudlet-installers.git C:\projects\installers
```

Comment:
Is this temporary until it can get merged into the main repository from keneanung's one?

Reply:
Yes. It's so I can test if this works or not.